From f5faaa2b24bf8edb7d8ecaa7390037c73c4d443d Mon Sep 17 00:00:00 2001
From: Laine Stump <laine@laine.org>
Date: Tue, 18 Oct 2011 12:40:41 -0400
Subject: [PATCH RHEL6.2 libvirt-snmp 1/3] allocate enough space for trailing
 NULL in string

This is a direct cherry-pick of upstream commit
17e238092ccaa3d6386ab668b651b3031f009e60, as a partial solution to:

  https://bugzilla.redhat.com/show_bug.cgi?id=732015
---
 src/libvirtGuestTable_data_get.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/libvirtGuestTable_data_get.c b/src/libvirtGuestTable_data_get.c
index 2430f9c..cc31e9e 100644
--- a/src/libvirtGuestTable_data_get.c
+++ b/src/libvirtGuestTable_data_get.c
@@ -201,7 +201,7 @@ libvirtGuestName_get( libvirtGuestTable_rowreq_ctx *rowreq_ctx, char **libvirtGu
     if (!rowreq_ctx->data.libvirtGuestName)
     	return MFD_SKIP;
 
-    len = strlen(rowreq_ctx->data.libvirtGuestName);
+    len = strlen(rowreq_ctx->data.libvirtGuestName) + 1;
     if ((NULL == (* libvirtGuestName_val_ptr_ptr )) ||
         ((* libvirtGuestName_val_ptr_len_ptr ) < len)) {
         /*
-- 
1.7.4.4

